conf/default_config.yaml

```diff
@@ -42,6 +41,15 @@ user_authentication:
   # WARNING: you should not change this after a system has been initialized as it may cause a mismatch in existing passwords
   hashed_passwords: ${ANCHORE_AUTH_ENABLE_HASHED_PASSWORDS}
 
+# This block is for defining archival rules around the maximum number of image analyses in the Anchore Deployment
```

Comment:
These rules are all configured via API calls, not configuration unless we extend the configuration to support any kind of rule. I don't want to split them up.

Reply:
Moved into rule schema